### DISCUSSION TITLE
qucs-s for OSX 10.14

### DISCUSSION THREAD

**Discussion by bnilsson11** [ra3xdh/qucs_s#882] [Q&A] [answered]
I need to install qucs-s on a OSX 10.14.6 machine.
Are there any archived qucs-s versions that could be installed on such a machine?

**Comment by dsm** [comment]
We release only qt6 build for macos if you need maybe we can build usign qt5 to support old macos release or you can build locally I only have a apple silicon mac and I used macos 14.

**Comment by dsm** [comment]
https://github.com/dsm/qucs_s/releases/download/continuous_build/Qucs-S-24.3.0-macOS-qt5-x86_64.dmg can you try this

**Comment by bnilsson11** [comment]
Thanks, but I installed the legacy version, it was ok for my purpose.

**Reply by dsm** [reply]
I build on macos 12 with qt5 this should be work your macos version if it works, i'll create a PR for this. I need your feedback.

**Comment by bnilsson11** [comment]
I could not run from Qucs-S-24.3.0-macOS-qt5-x86_64.dmg  qucs-s on my MacOS 10.14.6,

**Reply by dsm** [reply]
What error gives on program run.

**Reply by dsm** [reply]
I'll provide a solution for in cmake file we need to define macosx deployment target I set minimum 10.13 because of brew qt 5 package is needed macos 10.13. I setup vm for macos 10.14 I tested and pushed current branch.

**Reply by dsm** [reply]
https://github.com/dsm/qucs_s/actions/runs/10486612899/artifacts/1836405373 can you try this

**Comment by bnilsson11** [comment]
I will try it.

Den ons 21 aug. 2024 kl 12:10 skrev Muhammet Şükrü Demir <
***@***.***>:

**Reply by dsm** [reply, marked as answer]
this is the latest release and it worked 10.14 macos I tested in vm you can use it.
https://github.com/ra3xdh/qucs_s/releases/download/continuous_build/Qucs-S-24.3.0-macOSX-x86_64.dmg

**Comment by bnilsson11** [comment]
Thank you for the application, it runs on my 10.14.

However, there is something strange about the system library.
Some of the libs do not show up under the Libraries tab, as they are blacklisted.
Example: I am trying to simulate a circuit with LM334, which is located in VoltageReferences.lib.
If I remove this library from the blacklist, I get syntax errors at ”Calculate DC bias”:

ERROR: "REF1": Cannot load library component "LM334" from "/Applications/qucs-s.app/Contents/MacOS/share/qucs-s/library/VoltageReferences"
ERROR: Cannot simulate a text file!

This behavior does not occur in the ”modern” qucs-s 24.3.0, Voltagereferences.lib is not blacklisted, and LM334 works.

Also, in the GUI, the Delete and Backspace buttons do not work, I have to use the mouse right-click Delete to remove components.
There might be more, but this is what I have sen.

So the function of this qucs-s is not 100%

**Reply by dsm** [reply]
I normally fixed delete issue before and I tested in macos 14 it's worked. I recently switch to use macos and yes there might be bugs, you can open issue for them and I try to fixed it.

**Comment by bnilsson11** [comment]
Hi,

A comment about my ”DC Bias ” error.
I found that I had the wrong simulator selected, wrong path to Ngspice.
Once that was fixed both DC and TRANS situation work.

Delete/backspace is still not working, not a big problem as I can use the mouse right-click.
I have MacPro5,1 hardware.

**Reply by dsm** [reply]
I'll try to fix delete issue on older mac, can you create an issue for this?
